amgut/db/patches/0024.sql

```diff
@@ -0,0 +1,9 @@
+-- September 21, 2015
+-- remove question 146 from non-human surveys
+
+DELETE FROM ag.survey_answers
+WHERE survey_question_id = 146
+AND survey_id IN (
+    SELECT survey_id FROM ag.ag_login_surveys
+    JOIN ag.ag_consent USING (ag_login_id, participant_name)
+    WHERE parent_1_name = 'ANIMAL_SURVEY');
```

Comment:
Can this please be updated refer to the `surveys` table (as that's where the information that differentiates surveys is stored)?